src/System.IO.FileSystem.DriveInfo/src/System.IO.FileSystem.DriveInfo.csproj

```diff
@@ -107,9 +107,6 @@
     <Compile Include="$(CommonPath)\Interop\Linux\Interop.Libraries.cs">
       <Link>Common\Interop\Linux\Interop.Libraries.cs</Link>
```

Comment:
I think we can remove the OS-specific Interop.Libraries now since I don't see anything else that would P/Invoke to them. 

(That leaves only PathInternal.CaseSensitive/Insensitive preventing this from having a single unix implementation, which we can consider deciding at runtime. Fine to leave that to a follow-up change, though.)